app/src/main/java/org/stepik/android/view/step_quiz_choice/ui/view/QuizItemView.kt

```diff
@@ -16,17 +18,27 @@ class QuizItemView @JvmOverloads constructor(
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
 ) : FrameLayout(context, attrs, defStyleAttr) {
-
     companion object {
         const val MAX_CLICK_DURATION = 200
     }
 
     private var clickDuration: Long = 0
 
-    override fun dispatchTouchEvent(ev: MotionEvent?): Boolean {
+    override fun dispatchTouchEvent(ev: MotionEvent): Boolean {
         val webview = findViewById<ProgressLatexView>(R.id.itemChoiceLatex)
         val dispatched = webview.webView.dispatchTouchEvent(ev)
-        if (ev?.action == MotionEvent.ACTION_UP) {
+
+        val latexTextTouchEvent = MotionEvent.obtain(
+            ev.downTime,
+            ev.eventTime,
+            ev.action,
+            ev.x - DpPixelsHelper.convertDpToPixel(16f),
+            ev.y - DpPixelsHelper.convertDpToPixel(16f),
+            ev.metaState
+        )
+
+        findViewById<LatexSupportableEnhancedFrameLayout>(R.id.latex_text).dispatchTouchEvent(latexTextTouchEvent)
```

Comment:
It's better to store reference for this view instead of executing findViewById every time